src/lib.rs

```diff
@@ -148,10 +148,10 @@ pub mod keys;
 
 pub mod layer;
 pub mod layout;
+pub mod networking;
 
 #[cfg(feature = "appkit")]
 pub mod listview;
-pub mod networking;
```

Comment:
This is going to be, potentially, the dumbest question I've ever asked: was having `networking` immediately under `listview` (which is feature flagged for `appkit`) really _also_ feature flagging `networking`? I've always thought that the `cfg()` lines just covered the immediate next line.

Reply:
After I committed this, I also had the exact same question. Now that I've tested it a few times, I'm more confused than before. It wouldn't show up in the dinghy tests so I tried moving it and it showed up. I've just moved it back in my local environment and it works again.

Reply:
Well, I reverted that specific change back to we both understood as correct. The networking tests were/are there in CI from before I moved the line. Maybe it was in the tests were ran in the iOS simulator locally the whole time and I'm losing my mind.